virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -271,7 +271,7 @@ def create_reuse_external_snapshots(vm, pre_set_root_dir=None, skip_first_one=Fa
 
 
 def make_syslink_path_backing_files(pre_set_root_dir, volume_path_list, origin_image_format="qcow2",
-                                    created_image_format="qcow2", syslink_back_chain_lenghth=4):
+                                    created_image_format="qcow2", syslink_back_chain_lenghth=4, qemu_rebase=False):
```

Comment:
Please update docstring for 'qemu_rebase'. Thanks!

Reply:
Thanks yingshun, Done